### PR TITLE
Add reviewer attribution to confessions review channel

### PR DESCRIPTION
**Pull request by benaubin**

### CODE REVIEW

File: lib/main.ts [hunk 1]
```diff
@@ -285,13 +285,16 @@ export async function viewConfession(staging_ts: string, approved: boolean): Pro
...
+              approved ? `:true: Approved` : `:x: Rejected`
+            } by <@${reviewer_uid}> <!date^${Math.floor(Date.now() / 1000)}^{date_short_pretty} at {time}|${new Date().toISOString()}>.`;
             await web.chat.update({
                 channel: staging_channel,
                 ts: staging_ts,
                 text: '',
                 blocks: new Blocks([
                     new TextSection(new MarkdownText(`(staging) *${fields.id}* ${fields.text}`)),
-                    new TextSection(new PlainText(approved ? ":true: Approved" : ":x: Rejected")),
+                    new TextSection(new MarkdownText(statusText)),
```
Comment (anirudhb) [inline lib/main.ts:297]: No need to change to Markdown.

File: pages/api/interaction_work.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ interface BlockActionInteraction {
     type: 'block_actions';
     trigger_id: string;
     response_url: string;
-    user: string;
+    user: {id: string};
```
Comment (anirudhb) [inline pages/api/interaction_work.ts:31]:
```suggestion
    user: {
      id: string;
    };
```

**Review by anirudhb — COMMENTED** [review]
nits :)

File: lib/main.ts (hunk 1 above)
Comment (benaubin) [reply lib/main.ts:297]: incorrect. it doesn't let you do mentions or date format without markdown. that's the fix from earlier :)

**Review by anirudhb — APPROVED** [review]

**Pull request merged**